components/booking/HotelRooms/HotelRoomItem.tsx

```diff
@@ -41,67 +42,57 @@ const HotelRoomItem: React.FC<HotelRoomItemProps> = ({ room }) => {
   ).replace("{roomId}", room.hotel_room_id);
 
   return (
-    <div className="border rounded-lg p-4 shadow-md mb-4">
-      <h2 className="text-xl font-semibold mb-2 text-black">{room.name}</h2>
-
-      {/* Render the HTML description safely */}
-      <div
-        className="text-gray-700 mb-4"
-        dangerouslySetInnerHTML={{ __html: room.description }}
-      ></div>
-
-      {/* Price Per Night Section */}
-      {room.pricePerNight.amount > 0 ? (
-        <div className="text-lg font-medium text-black mb-4">
-          Price per night:{" "}
-          {`${room.pricePerNight.currency.symbol}
-          ${room.pricePerNight.amount} ${room.pricePerNight.currency.code}`}
-        </div>
-      ) : (
-        <div className="text-lg font-medium text-black mb-4">
-          Price per night: Unavailable
-        </div>
-      )}
-
-      {/* Room ID */}
-      <div className="text-sm text-gray-600 mb-4">
-        Room ID: {room.hotel_room_id}
+    <div className="flex flex-col gap-6 bg-base-200 rounded-box p-8 border border-primary shadow">
+      <div className="text-center text-base-content">
+        <h2 className="text-2xl font-semibold">{room.name}</h2>
+        <p className="text-lg text-base-content" dangerouslySetInnerHTML={{ __html: room.description }}>
```

Comment:
@ray-sjsu is `room.description` a string containing html? If not there was no need to dangerously set the inner html.
Instead we could simply render it as `<p>{room.description}</p>

Reply:
someone else commented on the code here saying it was to "ensure description" or something so i just kept it that way